cowprotocol/explorer
# [Fill Orders] Add Fill order table

# Summary

Closes #165 

Added Fills tab with fillable orders table.

<img width="1257" alt="Screen Shot 2022-10-28 at 15 29 17" src="https://user-images.githubusercontent.com/11525018/198707864-e254f603-2cec-4f12-9cc4-17ba21eef646.png">


# To Test

1. Open  `order` details page for a particular one.  i.e: `/orders/0x4a3927813f512fcc6ba710ee44d2a9204d21e997c5c122a60d1d4a2106fdd848042b32ac6b453485e357938bdc38e0340d4b927661c2639b`
       *  You'll see now both tabs for order overview and fill orders.

This is still WIP.
